docs: Edge Function docs (#18738)
supabase/supabase: apps/docs/pages/guides/functions/deploy.mdx

import Layout from '~/layouts/DefaultGuideLayout'

export const meta = {
  id: 'functions-deploy',
  title: 'Deploy to Production',
  description: 'Deploy your Edge Functions to your remote Supabase Project.',
  subtitle: 'Deploy your Edge Functions to your remote Supabase Project.',
  tocVideo: 'rzglqRdZUQE',
}

Once you have developed your Edge Functions locally, you can deploy them to your Supabase project.

## Login to the CLI

> If you're not yet logged into the Supabase CLI, you can do so by running:

Reviewer: ```suggestion
Log in to the Supabase CLI if necessary:
```

```bash
supabase login
```

<Admonition type="tip" label="CLI not installed?">

> Check out the [CLI Docs](/docs/guides/cli) to learn how to install the Supabase CLI on your local machine.

Reviewer: ```suggestion
See the [CLI Docs](/docs/guides/cli) to learn how to install the Supabase CLI on your local machine.
```
